tests/test_commands.py

```diff
@@ -70,12 +71,14 @@ def test_parse_command_merge():
     assert len(cmds) == 1
     assert cmds[0].name == "merge"
     assert cmds[0].bumpversion_mode == "patch"
-    cmds = list(parse_commands("/ocabot merge"))
+    cmds = list(parse_commands("/ocabot merge nobump"))
     assert len(cmds) == 1
     assert cmds[0].name == "merge"
-    assert cmds[0].bumpversion_mode is None
+    assert cmds[0].bumpversion_mode == "nobump"
+    with pytest.raises(RequiredOptionError):
+        list(parse_commands("/ocabot merge"))
     with pytest.raises(InvalidOptionsError):
-        list(parse_commands("/ocabot merge brol"))
+        list(parse_commands("/ocabot merge nobump brol"))
```

Comment:
Ok, but you still miss a test that "ocabot merge invalidoption" (with only one invalid option) raises an error of some sort :)

Reply:
``/ocabot merge invalidoption`` will not raise an invalid option, but a required option. This is tested here with ``/ocabot merge`` https://github.com/OCA/oca-github-bot/pull/110/files#diff-59262c12118a93348565a41ab7f6ce42R78

Reply:
> ocabot merge invalidoption will not raise an invalid Error, but a required Error

Sorry to insist, I must be missing something: I see a test with no options, but no test with exactly one invalid option, and we had such a test before, so I think it's good to keep it.

Reply:
OK. Sorry, I misread your comment. I'll add it.

Reply:
To be complete, my approach is to test each error separately.
In the ``/ocabot merge invalidoption``, there are two errors in fact
- ``invalidoption`` is not a valid option
- ``bump_mode`` is not set. 

I was thinking that testing wich error is raised first is not interesting, and can depend on the design and change, depending on perfomance reason, or developper choice. I mean, in Odoo for exemple, you can test that create a product without name will raise an error because name is required, and test that create a product without another required field also. But know which error is raised first adds nothing (IMHO) and can be not predictive in some cases BTW.

but well : not a problem, I'll add a test ;-)
Thanks.

Reply:
You can have both exception inherit from the same `OptionError` and test for that?